### PR TITLE
[19.1] Stories

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

[19.1] Stories

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3300/storybook

File: stories/documentation/listings/data-table/html&css/overflow-both.stories.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+interface OverflowBothStory {}
+
+export default {
+	title: 'Documentation/Listings/Data table/HTML&CSS/Overflow both',
+	argTypes: {},
+} as Meta;
+
+function getTemplate(args: OverflowBothStory): string {
+	return `<div class="dataTableWrapper" style="width: 30rem; height: 10rem">
```
Comment (vvalentin-lucca) [inline stories/documentation/listings/data-table/html&css/overflow-both.stories.ts:11]: A `white-space: nowrap` is needed for horizontal scrolling to take effect.

File: stories/documentation/listings/data-table/html&css/overflow-horizontal.stories.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+interface OverflowHorizontalStory {}
+
+export default {
+	title: 'Documentation/Listings/Data table/HTML&CSS/Overflow horizontal',
+	argTypes: {},
+} as Meta;
+
+function getTemplate(args: OverflowHorizontalStory): string {
+	return `<div class="dataTableWrapper" style="width: 30rem">
```
Comment (vvalentin-lucca) [inline stories/documentation/listings/data-table/html&css/overflow-horizontal.stories.ts:11]: Same here.

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3300/storybook

**Review by shajz — APPROVED** [review]

**Pull request merged**